### ISSUE TITLE
Functional specs download produces a JSON file

### ISSUE THREAD

**Issue by attaxia** [AmpersandTarski/RAP#114]
#### De situatie
When attempting to download the functional specifications document a download is initiated for a file called "script.json"

#### Wat er gebeurde er en wat had je verwacht?
I expected the file to be of the .doc or .docx extension.

#### Stappen om dit te reproduceren
1. Log in
2. Make any script
3. Click the "Func. spec + pictures" button
4. Click the "Functional specifications" download link

#### Screenshot / Video
<img width="486" alt="Screen Shot 2021-01-03 at 17 52 13" src="https://user-images.githubusercontent.com/4743303/103484106-9d344080-4dec-11eb-9eea-f61ef2f5f271.png">

#### Context
The JSON file actually contains Microsoft Word data and can be opened after renaming.
Using Firefox 84.0.1 on Mac OS Big Sur 11.1

**Comment by hanjoosten** [comment]
This is pretty strange. I expect this to have something to do with firefox / MacOS combination. I work on a windows machine, so I cannot reproduce this. @stefjoosten , do you have the same behaviour?

**Comment by hanjoosten** [comment]
@attaxia is this recent behavour? Did you generate and download functional spec documents before?

**Comment by attaxia** [comment]
This is recent behaviour since maybe one or two weeks. It worked before. It is not entirely MacOS related either, but may be Firefox related. Please note this behaviour in Linux on Firefox 82.0:

<img width="1297" alt="Screen Shot 2021-01-04 at 14 29 48" src="https://user-images.githubusercontent.com/4743303/103540156-91528800-4e99-11eb-8398-43dec67e4035.png">

It looks to me like the _content-type_ response header maybe incorrectly set to _application/json_.

**Comment by attaxia** [comment]
@hanjoosten It seems to be related to Firefox. Since updating Firefox in my Linux VM to 84.0 the same issue now happens on Linux.

Root cause seems to be still the incorrect content-type response header.
<img width="1510" alt="Screen Shot 2021-01-04 at 14 39 55" src="https://user-images.githubusercontent.com/4743303/103540875-b8f62000-4e9a-11eb-9ae8-a0dc92481220.png">

**Comment by attaxia** [comment]
@hanjoosten  Had to install Windows 10 to verify, but this does indeed also happen on Windows (in Firefox)
<img width="1136" alt="Screen Shot 2021-01-04 at 15 45 32" src="https://user-images.githubusercontent.com/4743303/103546991-16db3580-4ea4-11eb-8522-55eb5f5d1844.png">

**Comment by hanjoosten** [comment]
Thanks for the investigation! As a work around one could use Chrome. I am not very familiar with the technique of content-type response headers, so this might take quite a while until someone fixes this. @Michiel-s , are you familiar with this type of thing?

**Comment by Michiel-s** [comment]
Yes, I experienced it elsewhere also. Chromium browsers don't have this issue. But others do.
I fixed it a couple of weeks ago in the prototype framework. Fixed in v1.8.5 see [Releases](https://github.com/AmpersandTarski/prototype/releases)
Commit https://github.com/AmpersandTarski/prototype/commit/1735c1ad7f306a21f4e85b848851d36fa7090665

To fix this issue for RAP3, update the Dockerfile in `RAP4` and `RAP4USER` folders.

**Comment by stefjoosten** [comment]
Will do..

**Comment by stefjoosten** [comment]
> This is pretty strange. I expect this to have something to do with firefox / MacOS combination. I work on a windows machine, so I cannot reproduce this. @stefjoosten , do you have the same behaviour?

I have tried to reproduce on `rap.cs.ou.nl` using MacOS Catalina vs. 10.15.7 running Firefox vs. 85. This does not reproduce the situation. What I will do is upgrade to prototype vs. 1.8.5, check if things are still working on my laptop (which I expect they will because I cannot reproduce the bug), and install it on rap.cs.ou.nl.
